javascript-bank/bank.js

```diff
@@ -0,0 +1,34 @@
+/* exported Bank */
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if ((balance <= 0) || (typeof balance !== 'number') || (isNaN(balance)) || (!(Number.isInteger(balance)))) {
+    return null;
+  } else if (balance > 0) {
```

Comment:
Same thing here, no `else if` block necessary, this can just be done below the above conditional.